classes/middleware/PKPAuthenticateSession.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file classes/middleware/PKPAuthenticateSession.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPAuthenticateSession
+ *
+ * @ingroup middleware
```

Comment:
We actually don't need `@ingroup` now that we've got namespacing. (Also elsewhere)